core/mutations/index.js

```diff
@@ -0,0 +1,6 @@
+export { default as CREATE_ARTICLE } from "./CreateArticle";
```

Comment:
There is no need for this file as we will never import this files.

Reply:
I used an index to import the queries from /core/queries and mutations from /core/mutations instead of importing from several files

Reply:
We wont ever import this files directly, this code will be analyzed and a separate `core/generated/graphql.ts` will be generated. 
All our components import will go to that file like...
`import { useUserQuery } from 'core/generated/graphql`